README

```diff
@@ -65,7 +65,7 @@ tweak some of these options.
 4 - GUI
 -------
 You can enable the GTK GUI by setting GTK to 1 in the SConstruct build file. 
-GfceuX is deprecatiated in favor of the new GTK GUI.  You can disable the GTK GUI at
+GfceuX is depreciated in favor of the new GTK GUI.  You can disable the GTK GUI at
```

Comment:
I think the common term for this is "deprecated".

Reply:
fixed